src/components/BookItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function BookItem() {
+  return (
+
+    <li>
+      Book
+      <button type="button">Delete</button>
+    </li>
+
+  );
+}
+
```

Comment:
- [ ] It will be great to prepare the BookItem component to be ready for data as specified in the project specification. The way we can do this is by having the `BookItem` component receive book data as a prop and render the title and the author.

Example:
```
function BookItem({book}) {
  return (

    <li>
      Book.author
     Book.title
      <button type="button">Delete</button>
    </li>

  );
}
```

Reply:
Done